### DISCUSSION TITLE
How to reference a package(in local Monorepo) in a single frontend project？

### DISCUSSION THREAD

**Discussion by vaynevayne** [vercel/turborepo#5043] [Help] [answered]
Pre-content:

I have multiple frontend projects
The public parts of these front-end projects are extracted
Front-end projects and public parts are not in the same git repository
detail:

Use pnpm's workspace to manage these public parts, such as ui, utils:
- packages
   |- - ui
   |- - utils
- pnpm-workspace.ymal
Use in front-end projects A and B
import { Icon } from "@myPackage/ui";
import { isEmpty } from "@myPackage/utils";
...
question:

When I was developing A locally, I wanted to change the style of the Icon component in @myPackage/ui. Do I link @myPackage/ui and @myPackage/ui within A project?

Will this be a problem? Note that the dependency in @myPackage/ui is "@myPackage/utils": "workspace:*".

I haven't solved this problem: how to quickly debug and develop @myPackage/ui in project A

**Comment by anthonyshew** [comment, marked as answer]
This sounds like exactly the use case for a monorepo and Turborepo!

It may help to read our [Getting Started(https://turbo.build/repo/docs/getting-started/create-new) guide.
